flow/cmd/validate_mirror.go

```diff
@@ -25,6 +26,8 @@ var (
 func (h *FlowRequestHandler) ValidateCDCMirror(
 	ctx context.Context, req *protos.CreateCDCFlowRequest,
 ) (*protos.ValidateCDCMirrorResponse, error) {
+	ctx, cancelCtx := context.WithTimeout(ctx, time.Minute)
```

Comment:
this can take longer make it 5 mins?

Reply:
if validation takes that long it should have a status that gives a progress bar checking off which steps have passed

but for now we'll remove timeout on mirror validation